x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/ReferenceAttributeTests.java

```diff
@@ -14,18 +14,20 @@
 import org.elasticsearch.xpack.esql.core.type.DataType;
 
 public class ReferenceAttributeTests extends AbstractAttributeTestCase<ReferenceAttribute> {
-    public static ReferenceAttribute randomReferenceAttribute() {
+    public static ReferenceAttribute randomReferenceAttribute(boolean onlyRepresentable) {
         Source source = Source.EMPTY;
         String name = randomAlphaOfLength(5);
-        DataType type = randomFrom(DataType.types());
+        DataType type = onlyRepresentable
```

Comment:
Why this change?

Reply:
I needed it for one of the serialization tests - they needed to only receive representable types from a `ReferenceAttribute`. I believe they needed to make a value of that type.